packages/test/stochastic-test-utils/src/performActions.ts

```diff
@@ -115,6 +115,9 @@ export async function performFuzzActionsAsync<
 			operation = await generator(state)
 		) {
 			operations.push(operation);
+			if ("debug" in operation && operation.debug === true) {
```

Comment:
Hmm is the `in` check for TypeScript's sake, to avoid a cast?  I've not seen that pattern before.  It's not needed at runtime right?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

it is a JavaScript operator that typescript also uses for type narrowing.

i have a bunch of helpers in merge tree to do type safe narrowing via strongly typed type guards based on in:
https://github.com/microsoft/FluidFramework/blob/779ce38f5cb08c4ca248022ff9498e377e83f61d/packages/dds/merge-tree/src/segmentInfos.ts#L27